releases/UWP/README.md

```diff
@@ -60,11 +60,12 @@ When using Visual Studio this package requires Visual Studio 2015 Update 3 or la
     - System.Threading.Overlapped
     - System.Xml.XmlDocument
 
-### 5.2.2 (July 14, 2016)
-When using Visual Studio this package requires Visual Studio 2015 Update 3 or later.
+### 5.2.2 (June 27, 2016)
```

Comment:
Pulled that date from the release notes... I'll put down the NuGet one, it makes more sense to align with that.